### PR TITLE
Add API for updating User

### PR DESCRIPTION
**Pull request by steelbrain**
As advertised

### CODE REVIEW

File: src/users.ts [hunk 1]
```diff
@@ -144,13 +178,22 @@ export default function(zoomRequest: ReturnType<typeof request>) {
...
     ListUsers,
     GetUser,
     GetUserToken,
     CreateUser,
     DeleteUser,
-    UpdatePresenceStatus
+    UpdatePresenceStatus,
+    UpdateUser,
   };
 }
```
Comment (sedenardi) [inline src/users.ts:199]: Please maintain the newline at the end of files.

**Review by sedenardi — CHANGES_REQUESTED** [review]
Sorry for the delay. This looks good for the most part, just one very minor change.

**Comment by sedenardi** [comment]
Hi there, I'm going to close this unless the requested change is made.

**Comment by steelbrain** [comment]
Thanks for the reminder here @sedenardi.
The review comment mentions an EOF line, but I have checked on both, Github Web and in the files themselves, and there is an EOF line.

Can you please re-review? Thanks!

**Review by sedenardi — APPROVED** [review]

**Comment by sedenardi** [comment]
Merged and published to `npm` as version `1.7.0`. Thanks again for your contribution!

**Pull request merged**